meson.build

```diff
@@ -818,49 +823,36 @@ else
             dirs: tracker_prefix + '/bin',
             required: false,
         )
+        localsearch = find_program(
+            'localsearch',
+            dirs: tracker_prefix + '/bin',
+            required: false,
+        )
 
-        if tracker.found()
-            cdata.set(
-                'TRACKER_MANAGING_COMMAND',
-                '"tracker daemon"',
-            )
-            cdata.set(
-                'TRACKER_PREFIX',
-                '"'
-                + tracker_sparql.get_variable(pkgconfig: 'prefix')
-                + '"',
-            )
-            tracker_manager += 'tracker'
+        indexer_command = ''
+
+        if localsearch.found()
+            cdata.set('HAVE_TRACKER3', 1)
+            indexer_command = '"' + localsearch.full_path() + ' daemon"'
         elif tracker3.found()
             cdata.set('HAVE_TRACKER3', 1)
-            cdata.set(
-                'TRACKER_MANAGING_COMMAND',
-                '"tracker3 daemon"',
-            )
-            cdata.set(
-                'TRACKER_PREFIX',
-                '"'
-                + tracker_sparql.get_variable(pkgconfig: 'prefix')
-                + '"',
-            )
-            tracker_manager += 'tracker3'
+            indexer_command = '"' + tracker3.full_path() + ' daemon"'
+        elif tracker.found()
+            indexer_command = '"' + tracker.full_path() + ' daemon"'
         elif tracker_control.found()
-            cdata.set(
-                'TRACKER_MANAGING_COMMAND',
-                '"tracker-control"',
-            )
-            cdata.set(
-                'TRACKER_PREFIX',
-                '"'
-                + tracker_sparql.get_variable(pkgconfig: 'prefix')
-                + '"',
-            )
-            tracker_manager += 'tracker_control'
+            indexer_command = '"' + tracker_control.full_path() + '"'
         endif
-        tracker_found = (tracker.found() or tracker3.found() or tracker_control.found())
-        if not tracker_found
-            warning('tracker not found (required for Spotlight support)')
+        indexer_found = (
+            tracker.found()
+            or tracker3.found()
+            or tracker_control.found()
+            or localsearch.found()
+        )
+        if not indexer_found
+            warning('indexer not found (required for Spotlight support)')
```

Comment:
Might be better to explicitly call out that the build system looks for either Tracker or LocalSearch here, for clarity.